scheduler/bin/help-make-cluster

```diff
@@ -34,14 +38,35 @@ KUBECONFIG=${COOK_KUBECONFIG} $gcloud container clusters get-credentials "$CLUST
 # Add credentials to default kubeconfig for convenience purposes.
 $gcloud container clusters get-credentials "$CLUSTERNAME" --zone "$ZONE"
 
+echo "---- Upgrading the cluster master version to $VERSION"
+$gcloud container clusters upgrade "$CLUSTERNAME" --cluster-version $VERSION --zone "$ZONE" --quiet --master
+
+echo "---- Setting up cook priority classes in kubernetes"
+KUBECONFIG=${COOK_KUBECONFIG} kubectl create -f "${DIR}/priority-class-cook-workload.yaml"
+KUBECONFIG=${COOK_KUBECONFIG} kubectl create -f "${DIR}/priority-class-synthetic-pod.yaml"
```

Comment:
### Why?

These priority classes allow real Cook job pods to preempt Cook synthetic pods. This is important because otherwise, when new nodes become available, pending synthetic pods can race against matched Cook jobs and cause the matched Cook jobs to get failures.

Reply:
It's possible, but in our internal environments, we actually want this to be handled at the cluster creation level by the "infrastructure" side.